### DISCUSSION TITLE
Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended.

### DISCUSSION THREAD

**Discussion by caioagiani** [community/community#47890] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

I received this message on Github Copilot:

_Thank you for renewing your free access to GitHub Copilot. Your access to GitHub Copilot will be reviewed on 2024-01-29. GitHub Copilot checks eligibility monthly per our policy. No steps are needed on your end._

However, when I access https://github.com/settings/copilot I see that my access has expired.

**Comment by devenes** [comment, marked as answer]
Thank you for sharing your concern regarding your access to GitHub Copilot. Based on the message you received from GitHub, it appears that your free access to GitHub Copilot has been successfully renewed until 2024-01-29, and it is normal for the access to be reviewed on a monthly basis as per GitHub's policy.

It is possible that the expiration status you are seeing on the GitHub settings page could be a display issue or a delay in reflecting the updated status of your renewed access. However, it is important to note that the message you received from GitHub regarding the renewal of your access is the official communication and should be considered as the accurate information.

To reiterate, your free access to GitHub Copilot has been renewed until 2024-01-29, and no further action is required on your end.
